extension/content/components/recipes/form/filters/FallbackFO.js

```diff
@@ -42,6 +44,8 @@ export default function FallbackFO() {
   const notAnArray = "Filter Object(s) is not contained in an array";
 
   const validateFO = (value, err) => {
+    let action = ACTION_REMOVE_CLIENT_ERRORS;
+    let actionArgs = { name: "filter_object" };
```

Comment:
i think these should move down below to where the `if (foErrors) {` line is (line 60).